scripts/fluentui-publish/index.ts

```diff
@@ -100,7 +100,7 @@ export function fluentuiPostPublishValidation() {
 
   // sync fluent version
   execCommandSync(gitRoot, 'yarn', ['syncpack:fix']);
-  const gitStatus = execCommandSync(gitRoot, 'git', ['status', '--porcelain']);
+  const gitStatus = execCommandSync(gitRoot, 'git', ['status', '--porcelain', `\\*package.json`]);
```

Comment:
can you check in `execCommandSync` (line 81) if stderr is available when error is thrown ? would be nice to have the actual error message printed, so it's easier for someone who didn't work on this specifically to know what might be wrong by looking at pipeline logs ?

Reply:
Thanks, good point. Updated :)